client/cli/Cargo.toml

```diff
@@ -50,4 +50,6 @@ sp-version = { version = "5.0.0", path = "../../primitives/version" }
 tempfile = "3.1.0"
 
 [features]
+default = ["with-rocks-db"]
+with-rocks-db = ["sc-client-db/with-kvdb-rocksdb"]
 wasmtime = ["sc-service/wasmtime"]
```

Comment:
Hmm... nit: it might be better to have the feature named simply `rocksdb`? (to be consistent with the `wasmtime` feature, where we don't prepend `with-`)

Reply:
Changed to `rocksdb` for consistency in all similar places